src/CLFFT.jl

```diff
@@ -345,9 +345,9 @@ end
 
 dim(p::Plan) = begin
     res  = Int32[0]
-    size = Csize_t[0]
+    size = cl.CL_uint[0]
```

Comment:
Using `Csize_t` here resulted in an error on my system, as `Csize_t` translated to `UInt64` whereas `cl.CL_uint` translated to `UInt32`.